tests/config_test.go

```diff
@@ -195,11 +196,12 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				secretName = "secret-" + uuid.NewString()
 				secretPath = config.GetSecretSourcePath(secretName)
 
-				data := map[string]string{
-					"user":     "admin",
-					"password": "redhat",
-				}
-				tests.CreateSecret(secretName, testsuite.GetTestNamespace(nil), data)
+				secret := libsecret.New(secretName, map[string][]byte{
+					"user":     []byte("admin"),
+					"password": []byte("redhat"),
+				})
```

Comment:
There are other places then the files this PR changes that use libsecret.New, I think new PR is a better fit for what you suggest.
Do you mind if it will be done on a follow up PR?